First release of the MSPI API (#76006)
zephyrproject-rtos/zephyr: doc/releases/release-notes-3.7.rst

  * Added release API
  * Added support for mode selection via the device tree

* MSPI

>   * Add the new :ref:`MSPI(Multi-bit SPI) <mspi_api>` API, enabling support for advanced

Reviewer: ```suggestion
  * Add the new experimental :ref:`MSPI(Multi-bit SPI) <mspi_api>` API, enabling support for advanced
```
maybe better?

Author: Sure.

    SPI controllers and peripherals that typically require command, address and data phases.

>     The API now supports single wire SDR up to hex wires DDR communication in sync/async ways.

Reviewer: ```suggestion
    The API now supports from single wire SDR up to hex wires DDR communication in sync/async ways.
```
?

>   * Added APS6404L QPI pSRAM device driver that uses the MSPI API.

Reviewer: ```suggestion
  * Added APS6404L QPI pSRAM device driver.
```
Nit: I would probably just remove all 3 "that uses the MSPI API" from these 3 lines. It is implied from being in the list.

  * Added ATXP032 OPI NOR flash device driver that uses the MSPI API.
  * Added Ambiq Apollo3 MSPI controller driver that implements the MSPI API.
  * Added MSPI bus emulator under bus emulators to showcase the implementation of the MSPI API.
  * Added MSPI flash device emulator to showcase the use of the MSPI API and interfacing with
    MSPI bus controllers.

>   * Added mspi-async and mspi-flash example to showcase the use of MSPI device drivers.

Reviewer: ```suggestion
  * Added :zephyr:code-sample:`mspi-async` and :zephyr:code-sample:`mspi-flash` samples to showcase the use of MSPI device drivers.
```
maybe better with links to their docs?
